com.haysell.pos.yaml

```diff
@@ -0,0 +1,23 @@
+id: com.haysell.pos
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: haysell-pos
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=host
```

Comment:
why does it need "host" access?

Reply:
In PoS ( Poin of Sale ) applications there is a common practice file data exchange ( on software exports something, somewhere and some other software/component reads )

Because we just provide tools to configure file exchange/import/export we have no control/impact which location in client workstation will be configured as spool directory ( sorry for poor English((((